qa-tests-backend/src/test/groovy/ProcessVisualizationTest.groovy

```diff
@@ -87,20 +85,31 @@ class ProcessVisualizationTest extends BaseSpecification {
 
     @Tag("BAT")
     @Tag("RUNTIME")
-    // TODO(ROX-16461): Fails under AKS
-    @IgnoreIf({ Env.CI_JOB_NAME.contains("aks-qa-e2e") })
     def "Verify process visualization on kube-proxy"() {
         when:
         "Check if kube-proxy is running"
         def kubeProxyPods = orchestrator.getPodsByLabel("kube-system", ["component": "kube-proxy"])
+        def podOwnerIsTracked = false
+        for (pod in kubeProxyPods) {
+            if (orchestrator.ownerIsTracked(pod.getMetadata())) {
+                podOwnerIsTracked = true
+                break
+            }
+        }
         // We only want to run this test if kube-proxy is running
         Assume.assumeFalse(kubeProxyPods == null || kubeProxyPods.size() == 0)
 
         then:
         "Ensure it has processes"
+        def query = "Namespace:kube-system+Deployment:static-kube-proxy-pods"
+        // if the kube-proxy pod owner is tracked (e.g. DaemonSet), then sensor does not track the individual pods as
+        // `static-kube-proxy-pods` so we must assert against the DaemonSet `kube-proxy`
+        if (podOwnerIsTracked) {
+            query = "Namespace:kube-system+Deployment:kube-proxy"
```

Comment:
Nit: 
This looks a bit convoluted to do this instead of something like a ternary operation.
On the other hand, this allows us to have a comment exactly explaining why we use two different queries.
Up to you whether to change something about this or not :smile:

Reply:
I don't have a strong opinion on this. I sometimes prefer explicitly write the `if` because I think it's more readable in this case the comment might be enough.

Reply:
```
Violation: Rule=LineLength P=2 Line=99 Msg=[The line exceeds 120 characters. The line is 145 characters.] Src=[        def query = podOwnerIsTracked ? "Namespace:kube-system+Deployment:kube-proxy" : "Namespace:kube-system+Deployment:static-kube-proxy-pods"]
```
😱 I think I'm going to revert to having the `if`. Breaking the elvis would basically convert it in an ugly `if`. WDYT?

Reply:
Yes, let's keep it with the `if`, that's better than breaking the tenary up